bigtable/beam/helloworld/pom.xml

```diff
@@ -76,6 +80,18 @@
       <artifactId>truth</artifactId>
       <version>1.1.3</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <!-- guava is required override the guava from the shared-configuration which causes build issues -->
```

Comment:
I would reword this slightly:

"manually declare guava to override version from shared-configuration"